site/gdocs/components/Chart.tsx

```diff
@@ -105,7 +111,22 @@ export default function Chart({
                     border: "0px none",
                     height: d.height,
                 }}
-            />
+            >
+                {isExplorer ? (
+                    <noscript>
+                        <p className="align-center">
+                            Interactive visualization requires JavaScript.
+                        </p>
+                    </noscript>
+                ) : (
+                    url.slug && (
+                        <a href={resolvedUrl} target="_blank" rel="noopener">
+                            <GrapherImage slug={url.slug} alt={d.title} />
```

Comment:
I think here you'll need to extract the slug from `resolvedUrl` because if we've updated the slug since publishing this article then we'll no longer be baking a static version at the old URL.

<img width="536" alt="image" src="https://github.com/owid/owid-grapher/assets/11844404/0821fa67-7baf-400f-9ea1-60898cd5ec12">

The 404'ing route: https://assets.ourworldindata.org/grapher/exports/share-living-in-extreme-poverty-basic-needs-estimate.svg

The static export is actually here: https://assets.ourworldindata.org/grapher/exports/share-of-population-living-in-extreme-poverty-cost-of-basic-needs.svg

Reply:
Good catch, thanks!